specification/trustedsigning/TrustedSigning/tspconfig.yaml

```diff
@@ -27,10 +27,10 @@ options:
     flavor: azure
   "@azure-tools/typespec-ts":
     package-dir: "azure-developer-trustedsigning-rest"
-    generateMetadata: true
-    generateTest: true
-    packageDetails:
-      name: "@azure-rest/developer-trusted signing"
+    generate-metadata: true
+    generate-test: true
+    package-details:
+      name: "@azure-rest/developer-trustedsigning"
```

Comment:
package name change refer to : https://github.com/Azure/azure-rest-api-specs/pull/30664#discussion_r2013995262